app/views/api/messages/_message.json.jbuilder

```diff
@@ -0,0 +1,15 @@
+json.id message.id
+json.from_user_id message.from_user_id
+json.to_user_id message.to_user_id
+json.title message.title
+json.sent_on message.sent_on.xmlschema
+
+if current_user.id == message.from_user_id
+  json.from_user_visible message.from_user_visible
```

Comment:
Should this just be `visible` in the API response? or maybe even invert it and call it `deleted` instead?

Reply:
Updated to `deleted`